src/test/java/HomeTest.java

```diff
@@ -0,0 +1,121 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+import pages.HomePage;
+import pages.LoginPage;
+
+import java.util.List;
+
+public class HomeTest extends BaseTest {
+
+
+    @Test
+    public void hoverOverPlayBtn() throws InterruptedException {
+        //Login;
+        provideEmail("kaflimeerim@gmail.com");
+        providePassword("te$t$tudent");
+        loginToKoel();
+        Thread.sleep(2000);
```

Comment:
Please, delete `throws InterruptedException`

Thread. sleep is bad! It blocks the current thread and renders it unusable for further work.
Using Thread. sleep() frequently in an automation framework is not a good practice.

Please use the Explicit Wait command, the WebDriver is directed to wait until a certain condition occurs before proceeding with executing the code.